packages/babel/src/transformation/file/index.js

```diff
@@ -578,7 +591,7 @@ export default class File {
   }
 
   /**
-   * [Please add a description.]
+   * If this should be ignored, pass it through, otherwise run callback.
```

Comment:
What's a good term for referring to the return value of `makeResult()`? "Output object"? "Result object"?

How about this?:

> Return output object containing `code` (if ignored), or callback result.
> Pass `code` through to output untransformed if File is ignored.

How about this for `shouldIgnore()`?:

> Input should not be transformed?
> @return Exclude from transformation if `true`.

Or just:

> Should input be excluded from transformation.